app/api/api_v1/routers/admin.py

```diff
@@ -845,7 +845,7 @@ def validate_climate_laws_urls(
     db=Depends(get_db),
     current_user=Depends(get_current_active_superuser),
 ):
-    """Validate that all documents that have climate-laws source urls have documents stored in our cdn."""
+    """Validate all documents have climate-laws source urls are our cdn."""
```

Comment:
I mean, there's shortening & there's creating gibberish :joy: 
```suggestion
    """Validate all documents with climate-laws source urls are in our cdn."""
```